macros/alter_table_properties.sql

```diff
@@ -550,6 +550,43 @@ ALTER TABLE airdrop_optimism.addresses SET TBLPROPERTIES ('dune.public'='true',
                                                 'dune.data_explorer.contributors'='["msilb7"]');
 {% endset %}
 
+{% set transfers_optimism_eth %}
+ALTER TABLE transfers_optimism.eth SET TBLPROPERTIES ('dune.public'='true',
+                                                'dune.data_explorer.blockchains'='["optimism"]',
+                                                'dune.data_explorer.category'='abstraction',
+                                                'dune.data_explorer.abstraction.type'='sector',
+                                                'dune.data_explorer.abstraction.name'='transfers',
+                                                'dune.data_explorer.contributors'='["msilb7", "chuxinh"]');
+{% endset %}
+
+{% set uniswap_v3_optimism_pools %}
+ALTER TABLE uniswap_v3_optimism.pools SET TBLPROPERTIES ('dune.public'='true',
```

Comment:
from what i can tell in the dbt project file (defaults to view) and model itself (no override in config), this object is a view, not table. if that's expected, this should be updated to ALTER VIEW